float64 vMax
bool setToDefault
---
